PhpAmqpLib/Connection/AMQPStreamConnection.php

```diff
@@ -33,10 +33,10 @@ public function __construct(
         $login_response = null,
         $locale = 'en_US',
         $connection_timeout = 3.0,
-        $read_write_timeout = 3.0,
+        $read_write_timeout = 130.0,
         $context = null,
         $keepalive = false,
-        $heartbeat = 0,
+        $heartbeat = 60,
```

Comment:
Currently, we are facing an issue regarding this change and we think this caused a BC break!